README.rst

```diff
@@ -179,6 +182,61 @@ Additionally Supported Parameters
 |                 |                                                                             |
 |                 | Default to **None** meaning above message would not be added to the email.  |
 +-----------------+-----------------------------------------------------------------------------+
+|       type      | Set values of other parameters based on predefined setting                  |
+|                 | ``OPENWISP_NOTIFICATION_TYPES``                                             |
+|                 |                                                                             |
+|                 | Defaults to **None** meaning you need to provide other arguments.           |
++-----------------+-----------------------------------------------------------------------------+
+
+Settings
+--------
+
+``OPENSWISP_NOTIFICATION_TYPES``
```

Comment:
I don't think we should make this a setting, it will be managed by register and unregister, right? So we should explain how to use those functions I think, because just changing the setting will have no effect, users of this module will have to use the new notification types in their code, so this is more a developer feature than just a setting. Don't you think?